src/table/src/TableRow.js

```diff
@@ -99,7 +99,7 @@ const TableRow = memo(
     const height = rest.height || themeHeight
 
     return (
-      <TableRowProvider height={height}>
+      <TableRowProvider value={height}>
```

Comment:
Would be good to verify that this doesn't introduce any issues, given that it's been like this for a long time and things "work"